crates/services/upgradable-executor/src/executor.rs

```diff
@@ -153,7 +153,8 @@ impl<S, R> Executor<S, R> {
         ("0-36-0", 11),
         ("0-37-0", 12),
         ("0-37-1", 13),
-        ("0-38-0", LATEST_STATE_TRANSITION_VERSION),
+        ("0-38-0", 14),
+        ("0-39-0", LATEST_STATE_TRANSITION_VERSION),
```

Comment:
Question mostly for me to understand: do we update the genesis state transition version at every release? Or would there be cases when this does not need to be done (e.g. if we don't change the executor at all between two releases)?

Reply:
We do this for each release, but yeah, it is a problem in the case when we need to update only off-chain logic. 

It would be nice if all off-chain logic lived in another place, in this case we don't need to release `fuel-core` often itself